client/src/main/java/io/pravega/client/stream/impl/ReaderGroupImpl.java

```diff
@@ -174,6 +173,7 @@ public CompletableFuture<Checkpoint> initiateCheckpoint(String checkpointName, S
 
         });
         if (!canPerformCheckpoint) {
+            log.warn(rejectMessage);
```

Comment:
Dont we need any log here to look for this warn?

Reply:
Removed